Sources/Plasma/CoreLib/HeadSpin.h

```diff
@@ -69,17 +69,29 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
     struct HINSTANCE__; typedef struct HINSTANCE__ *HINSTANCE;
 
     typedef HWND hsWindowHndl;
+    typedef HWND hsDisplayHndl;
     typedef HINSTANCE hsWindowInst;
     typedef HINSTANCE HMODULE;
     typedef HMODULE hsLibraryHndl;
     typedef long HRESULT;
     typedef void* HANDLE;
 #elif HS_BUILD_FOR_MACOS
+    // Same note as Windows above - would rather not forward declare but I don't want to
+    // import Foundation or CoreGraphics
+#if HS_BUILD_FOR_IOS
+    // Exception - iOS doesn't support CGDirectDisplayID.
+    // It has UIScreen but that's a Cocoa type.
+    typedef void* CGDirectDisplayID;
+#else
```

Comment:
I think this should be `#endif` not else? (otherwise hsDisplayHndl never actually gets defined for iOS)

or else CGDirectDisplayID should be hsDisplayHndl